src/lib/bootstrap.js

```diff
@@ -32,12 +36,13 @@ ga('send', 'pageview');
 // site code. This includes Shadow DOM.
 const browserSupport = 'noModule' in HTMLScriptElement.prototype;
 if (browserSupport) {
-  function prepare() {
+  const prepare = () => {
```

Comment:
just curious, is this gst style?

Reply:
It would want a function declared at the top level, rather than in an `if` statement.